migrations/sql/V1.35__add_user_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE minespace_users (
+    id serial PRIMARY KEY,
+    keycloak_guid uuid,
+    email character varying(100) NOT NULL, 
```

Comment:
Can you please add comments to these tables describing their use? ie. Email is intended to be the unique BCEID email being used as our primary identifier, and that mds_mine_access is intended to represent any ability to access data related to these mines.